markdown/summary.Rmd

```diff
@@ -0,0 +1,94 @@
+---
+output: 
+  word_document:
+    reference_docx: "summary-template.docx"
+---
+
+```{r, echo=FALSE, message=FALSE, warning=FALSE,error=FALSE}
+
+#Create a pseudo chart
+library(ggplot2)
+
+Year <- c(rep("2014", times = 8), rep("2015", times = 8))
+Month <- rep(c(4, 5, 6, 7, 8, 9, 10, 11), times = 2)
+Count <- c(800, 820, 840, 860, 880, 1000, 1100, 1200, 
+           800, 840, 860, 880, 1000, 1100, 1200, 1300)
+
+ChartData <- data.frame(Year, Month, Count)
+
+chart <- ggplot(data = ChartData, aes(x = Month, y = Count, group = Year, color = Year)) +
+  geom_line(size = 1) + 
+  geom_point(size = 3) +
+  theme(plot.title = element_text(hjust = 0.5, face = "bold", size = 11),
+    panel.background = element_blank(),
+    panel.grid.major = element_blank(), 
+    panel.grid.minor = element_blank(),
+    axis.text.x = element_text(angle = 90),
+    axis.title.y = element_text(margin = margin(0,10,0,0)),
+    axis.line = element_line(colour="grey"),
+    axis.ticks = element_line(colour="grey")) +
+  xlab("Month") + ylab("Count") +
+  ggtitle("Percentage of last six months of life spent at home or in a community setting; 2010/11 - 2018/19(1)")
+
+ggsave("charts/chart1.png", plot = chart, width = 8, height = 4, device = "png", dpi = 300)
+
+#Create contact information
+contact1 <- c("**Federico Centoni**", "Information Analyst", 
+              "0131 314 1532", "federico.centoni@nhs.net")
+contact2 <- c("**Alice Byers**", "Senior Information Analyst", 
+              "0131 314 1725", "alice.byers@nhs.net")
+contact3 <- c("**Aghimien Iyayi-Igbinovia**", "Principal Information Analyst", 
+              "0141 207 1616", "aghimien.iyayi-igbinovia@nhs.net")
+
+contact <- data.frame(contact1, contact2, contact3)
+```
+
+# Percentage of End of Life Spent at Home or in a Community Setting (Provisional)
```

Comment:
We need to break the line after "or" as the title is too long. I tried:
# Percentage of End of Life Spent at Home or\ 
# in a Community Setting (Provisional)

It works but the line spacing is too big. Maybe needs to adjust the line spacing manually afterwards.

Reply:
I see what you mean - will have a google but if no solution, manually fixing the spacing won't be the end of the world.